pkg/drivers/kic/kic.go

```diff
@@ -102,8 +102,13 @@ func (d *Driver) Create() error {
 		params.IP = ip.String()
 	}
 	drv := d.DriverName()
+
 	listAddr := oci.DefaultBindIPV4
-	if oci.IsExternalDaemonHost(drv) {
+	if d.NodeConfig.ListenAddress != "" && d.NodeConfig.ListenAddress != listAddr {
+		out.WarningT("Listening to {{.listenAddr}}. This is not recommended and can cause a security vulnerability. Use at your own risk",
```

Comment:
add an additional line above (in addition to the warnning bellow) to make it clear.

out.Infof(style.Tip, "minikube is not meant for production use. you are opening non-local traffic")

Reply:
Added it, since Infof does not allow styles, I opted to just have the message.

Reply:
Changed it now to be above the warning message as well as to be a styled as a tip using `Step()`.